proposal/color-4-new-spaces.md

```diff
@@ -548,7 +549,7 @@ The known color spaces and their channels are:
       > Percentages `[0%,100%]` map to the `[0,100]` range.
 
   * `a`, `b`:
-    * gamut: un-bounded
+    * gamut: un-bounded, scalable
```

Comment:
Is it even wroth explicitly listing channels as "scalable", given that they're now scalable if and only if they're not polar-angle?